### PR TITLE
Uml 1154 add lpa triage page

### PR DESCRIPTION
**Pull request by GemTay**
# Purpose

Completes the triage page to allow users to to take to correct path to add an lpa to their account

Fixes UML-1154

## Approach

Created a form with the radio buttons and validation message. Setup redirects to add lpa/ask for activation key page based on radio button selected. Created new handler/route/twig template for the "Ask for an activation key" page, purely to show user flow to that page if radio button "No" was selected

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [x] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [x] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/app/src/Actor/src/Handler/AddLpaTriageHandler.php [hunk 1]
```diff
@@ -35,6 +34,36 @@ public function __construct(
...
+    {
+        $form = new AddLpaTriage($this->getCsrfGuard($request));
+        $form->setData($request->getParsedBody());
+
+        if ($form->isValid()) {
+            $selected = $form->getData()['activation_key_triage'];
+
+            if ($selected === 'Yes') {
+                return $this->redirectToRoute('lpa.add-by-code');
+            } elseif ($selected === 'No') {
```
Comment (GemTay) [inline service-front/app/src/Actor/src/Handler/AddLpaTriageHandler.php:59]: @cooperaj I'm unsure how strict I should be here. Is the elseif needed or can anything other than "Yes" just redirect the user to the other page?

File: service-front/app/src/Actor/src/Form/AddLpaTriage.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    public function getInputFilterSpecification(): array
+    {
+        return [
+            'activation_key_triage' => [
+                'required'   => true,
+                'validators' => [
+                    [
+                        'name'                   => NotEmpty::class,
+                        'break_chain_on_failure' => true,
```
Comment (cooperaj) [inline service-front/app/src/Actor/src/Form/AddLpaTriage.php:44]: Technically not needed as it's the only item in the chain.

File: service-front/app/src/Actor/src/Form/AddLpaTriage.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                        'options'                => [
+                            'messages'           => [
+                                NotEmpty::IS_EMPTY => 'Select if you have an activation key to add the LPA',
```
Comment (cooperaj) [inline service-front/app/src/Actor/src/Form/AddLpaTriage.php:47]: Am wondering if it's worth starting the migration to PHP translation extraction for this?

**Review by cooperaj — APPROVED** [review]
Couple of things that maybe need a chat but still good to go.

File: service-front/app/src/Actor/src/Form/AddLpaTriage.php (hunk 2 above)
Comment (GemTay) [reply service-front/app/src/Actor/src/Form/AddLpaTriage.php:44]: Ah good spot, i'll remove it on my next ticket to save having to get a re-review!:)

File: service-front/app/src/Actor/src/Form/AddLpaTriage.php (hunk 3 above)
Comment (GemTay) [reply service-front/app/src/Actor/src/Form/AddLpaTriage.php:47]: Yeah, manually adding the errors to form-error-messages.html.twig isn't a good long term solution as we're going to forget to add new error messages or update them at some point

**Pull request merged**